app/src/components/typography.css

```diff
@@ -0,0 +1,66 @@
+.typography--variant-h1 {
```

Comment:
Why did you delete the `module` ?

Reply:
Cuz styles work differently when you use .module.css. In this case you have to import "styles" variable and put "styles.className" in each component. We just want to import all CSS in the whole component

Reply:
Okay :+1: